js/huobi.js

```diff
@@ -4618,6 +4619,121 @@ module.exports = class huobi extends Exchange {
         return this.filterByArray (result, 'symbol', symbols);
     }
 
+    async fetchBorrowInterest (code = undefined, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        if (since !== undefined) {
+            request['start-date'] = this.yyyymmdd (since);
+        }
+        if (limit !== undefined) {
+            request['size'] = limit;
+        }
+        let market = undefined;
+        let method = 'privateGetCrossMarginLoanOrders';
+        if (symbol !== undefined) {  // Isolated
```

Comment:
@samgermain @carlosmiei i think it should not depend on whether the symbol is present or not, it should rather depend on the marginType setting, no?

- https://github.com/ccxt/ccxt/blob/master/js/huobi.js#L1937

Reply:
@kroitor Yes seems semantically better, but since @samgermain wrote this bit I'd wait for his take

Reply:
@carlosmiei i mean we're already doing this i all other methods, so i think it makes sense. Especially, if their API can return mixed margin coins from the isolated-margin endpoint: https://huobiapi.github.io/docs/spot/v1/en/#search-past-margin-orders-isolated it doesn't look like it requires a symbol.

Reply:
@kroitor I've just rewritten it accordingly

Reply:
@kroitor This is how it is for every other exchange, if they select `isolated` they will need to provide a market symbol anyway, so checking for the margin type seems unnecessary

Reply:
Symbol is a required argument for huobi, so there's no need to check for margin type. If they've provided a symbol, they want the interest for a specific isolated margin wallet, if they haven't provided a symbol, then there's no way to know which isolated market they want, so we default to cross

There's just no reason to check the margin type, we have all the information we need if they provide a symbol